docs/modules/reference/pages/daemons/daemon-config-files/queued.adoc

```diff
@@ -1,15 +1,16 @@
+
 [[ref-daemon-config-files-queued]]
 = Queued
+
 Handles scheduling tasks to run in the future.
+
 .Queued log file overview
 [options="header"]
 [cols="2,3"]
-
 |===
 | File
 | Description
 
 | queued.log
 | Logs relating to Queued server events.
```

Comment:
Was going to lower case this, but it is one case where it isn't clear it's a daemon since lower case it could be "queued".

Reply:
That's what I was thinking too. I'm going to include this as an example of an exception to our rule in the Style Guide. :)